SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x7b1c1b

During execution of a job, I get:

```
Failed to find the SystemVssConnection Endpoint, try to finish job as failedpanic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x58 pc=0x7b466e]

goroutine 177657 [running]:
github.com/ChristopherHX/github-act-runner/actionsrunner.runJob.func1.2()
        /home/runner/work/github-act-runner/github-act-runner/actionsrunner/runner.go:492 +0xee
created by github.com/ChristopherHX/github-act-runner/actionsrunner.runJob.func1 in goroutine 177597
        /home/runner/work/github-act-runner/github-act-runner/actionsrunner/runner.go:487 +0xb05
```

Then trying to restart I get immediately:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x7b1c1b]

goroutine 23 [running]:
github.com/ChristopherHX/github-act-runner/protocol.(*VssConnection).FinishJob(...)
        /home/runner/work/github-act-runner/github-act-runner/protocol/connection.go:375
github.com/ChristopherHX/github-act-runner/actionsrunner.(*RunRunner).Run.func5.2()
        /home/runner/work/github-act-runner/github-act-runner/actionsrunner/runner.go:174 +0x11b
created by github.com/ChristopherHX/github-act-runner/actionsrunner.(*RunRunner).Run.func5 in goroutine 22
        /home/runner/work/github-act-runner/github-act-runner/actionsrunner/runner.go:172 +0x54f
```

After a couple of attempts it will eventually start.

This makes my runner go down every now and then.

--- a/actionsrunner/runner.go
+++ b/actionsrunner/runner.go
@@ -616,7 +616,8 @@ func runJob(runnerenv RunnerEnvironment, joblock *sync.Mutex, vssConnection *pro
 }
 
 func renewJob(jobctx context.Context, runServiceURL string, jobreq *protocol.AgentJobRequestMessage,
-	con *protocol.VssConnection, instance *runnerconfiguration.RunnerInstance) error {
+	con *protocol.VssConnection, instance *runnerconfiguration.RunnerInstance,
+) error {
 	if runServiceURL != "" {
 		jobVssConnection, _, conErr := jobreq.GetConnection("SystemVssConnection")
 		if conErr != nil {

